src/common/flowConfig.ml

```diff
@@ -380,6 +382,8 @@ module OptionsParser = struct
     options
 end
 
+let file_extension = Str.regexp "^\\(\\.[^ \t\\.]+\\)$"
```

Comment:
If I'm reading this correctly, this excludes something like ".es6.js" (two dots) -- is there a reason you restricted this?